app/generate.js

```diff
@@ -71,17 +84,39 @@ async function generateSections(renderedComponents) {
   return retArray;
 }
 
-function generateImages(theme, images) {
+function generateImages(theme, images, defaultLang) {
   const retArray = [];
   const imageTemplate = template(fs.readFileSync(`./templates/${theme}/image.tpl.html`, 'utf8'));
 
   for (let i = 0; i < images.length; i += 1) {
     const image = images[i];
-    retArray.push({ html: imageTemplate(image), id: image.id, file: image.file });
+    image.webRoot = defaultLang.id === image.lang.id ? './' : '../';
+    retArray.push({
+      html: imageTemplate(image),
+      id: image.id,
+      file: image.file,
+    });
   }
   return retArray;
 }
 
+function generateLangMenu(org, pages) {
+  const { defaultLang, theme } = org;
+  const langs = [];
+  const webRoot = '/';
+  for (let i = 0; i < pages.length; i += 1) {
+    const { lang } = pages[i];
+    if (lang.id !== defaultLang.id && !langs.find(element => element.id === lang.id)) {
+      langs.push(lang);
+    }
+  }
+  if (langs.length === 0) {
```

Comment:
I don't expect to see the menu if there's just one language, so maybe `langs.length <= 1` would be more appropriate?